src/Routing/Filter/AssetFilter.php

```diff
@@ -118,8 +117,7 @@ protected function _deliverAsset(Request $request, Response $response, $assetFil
 			$response->header('Content-Length', filesize($assetFile));
 		}
 		$response->cache(filemtime($assetFile));
-		$response->send();
-		ob_clean();
+		$response->sendHeaders();
 		readfile($assetFile);
 		if ($compressionEnabled) {
 			ob_end_flush();
```

Comment:
For two reasons:
1- You've made the last change on this filter.
2- The causes of this problem can have a lot of reasons, and you have good experience in this case.
So I think you are the best choice for notification of the problem and solve it.

I have a request of you:
Since I do not know the English language well. i am not understand very rules of community, Please notify me out whenever I was wrong. THX :)